tests/internal/fuzzers/flb_json_fuzzer.cpp

```diff
@@ -0,0 +1,20 @@
+#include <stdint.h>
+#include <stdlib.h>
+
+extern "C" {
+    int flb_pack_json(char*, int, char**, size_t*, int*);
+}
+
+extern "C" int LLVMFuzzerTestOneInput(unsigned char *data, size_t size)
+{
+	// json packer
+	char *out_buf = NULL;
+	size_t out_size;
+	int root_type;
+	flb_pack_json((char*)data, size, &out_buf, &out_size, &root_type);
```

Comment:
please validate the return value of flb_pack_json

Reply:
Done!